JavaErickHernandez/Day3/RansomNote.java

```diff
@@ -0,0 +1,87 @@
+package bootcamp;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Scanner;
+
+public class RansomNote {
+	static String wordsMagazine = "Enter the number of words in the magazine: ";
```

Comment:
What is the reason for not putting these variables as "final"? if you put "final" you avoid to any other person can change this variables in other par of the code.